src/components/RegType/StepsList.jsx

```diff
@@ -29,14 +32,13 @@ function StepsList(props) {
     <path d="M21.8955 27.3984H39.2424V29.5668H21.8955V27.3984Z" fill="#11385B"/>
     <path d="M21.8955 32.8203H30.5689V34.9887H21.8955V32.8203Z" fill="#11385B"/>
     </svg>
-    
+
 
     return (
         <>
-
-            <h2>{content.process_heading}</h2>
-            <p>{content.process_parag}</p>
-
+            <div dangerouslySetInnerHTML= {{__html: contentHeading}}/> {/* TODO: Need to style each paragraph */}
```

Comment:
We don't need html for the heading field. We should have an h2 here. Maybe this needs to be fixed on the CMS side.

Reply:
Yes this should be fixed on the CMS side. I'll change the json we have committed for now.